src/nvim/shada.c

```diff
@@ -3482,6 +3489,8 @@ static inline char *get_converted_string(const vimconv_T *const sd_conv,
           (tgt) = aetv.vval.v_list; \
         } \
       } while (0)
+// Workaround for silencing linter.
+#define ELSE else
```

Comment:
This is unpleasant. Linting rule should be disabled if it's going to cause this kind of cascading workaround.